src/elm/Page/Account.elm

```diff
@@ -285,18 +286,37 @@ view _ _ _ model _ =
 
 
 viewSidebar : Model -> Html Msg
-viewSidebar _ =
-    Ui.Section.view
-        [ B.init "Logg ut"
-            |> B.setIcon (Just Icon.logout)
-            |> B.setOnClick (Just Logout)
-            |> B.tertiary
-        , B.init "Slett konto"
-            |> B.setIcon (Just Icon.delete)
-            |> B.setDisabled True
-            |> B.setOnClick (Just Logout)
-            |> B.tertiary
-        ]
+viewSidebar model =
+    let
+        phoneNumber =
+            model.profile
+                |> Maybe.map .phone
+                |> Maybe.withDefault "<Telefon nummer her>"
+
+        subject =
+            "Slett meg som bruker"
+
+        body =
+            "Jeg vil slettes som bruker av nettbutikk og AtB systemene. Brukeren min er ved telefonnummer: " ++ phoneNumber
+
+        deleteLink =
+            "mailto:kundeservice@atb.no"
+                ++ Url.toQuery
+                    [ Url.string "body" body
+                    , Url.string "subject" subject
+                    ]
+    in
+        Ui.Section.view
+            [ B.init "Logg ut"
+                |> B.setIcon (Just Icon.logout)
+                |> B.setOnClick (Just Logout)
+                |> B.tertiary
+            , B.init "Slett konto"
+                |> B.setIcon (Just Icon.delete)
```

Comment:
Skulle dette ikonet hatt en indikasjon om at man (mest sannsynligvis) åpner et nytt vindu?

Reply:
Usikker på hvordan det blir med en `mailto:` men burde i alle fall indikere med en title som er mer beskrivende. Det er noen par andre target _blank lenker vi har som burde markeres strengt tatt.

Reply:
Ja, de burde også hatt et ikon som viser tydelig at det åpnes noe i et nytt vindu/fane. For web-baserte mail-klienter så åpnes det normalt et nytt vindu/fane, og ellers så er det en context switch til en annen applikasjon enn browseren. En beskrivende tekst er bedre enn ingenting iafl.